dash/include/dash/memory/GlobHeapMem.h

```diff
@@ -429,6 +429,23 @@ class GlobHeapMem
     return unit_local_size;
   }
 
```

Comment:
... and `sizeof(element_type)` is a power of two, so it should become a left-shift, right.